app/components/nextflow_component.html.erb

```diff
@@ -0,0 +1,57 @@
+<h1
+  class="
+    mb-4
+    text-2xl
+    font-extrabold
+    leading-none
+    tracking-tight
+    text-gray-900
+    md:text-5xl
+    lg:text-4xl
+    dark:text-white
+  "
+>
+  <%= schema["title"] %>
+</h1>
+<p class="mb-4 text-lg font-normal text-gray-500 lg:text-xl dark:text-gray-400">
+  <%= schema["description"] %>
+</p>
+<form>
+  <% schema["definitions"].each do |_item, definition| %>
+    <%- if definition["properties"]
+      .values
+      .filter { |property| property.key?("hidden") }
+      .length != definition["properties"].values.length %>
```

Comment:
**nitpick (blocking):** I think this if statement should only trigger when the length of non hidden values is > 0 instead

Reply:
I disagree. If it triggers at > 0 that means if there is 1 out of 5 fields hidden then the whole thing would be hidden - which we do not want.

Reply:
I said non hidden values > 0, which would mean that if 1 out of 5 fields is hidden then it would still show the section with the non hidden value.

Reply:
In this case, I guess what you have achieves the same thing that I was asking for, I just interpreted the code incorrectly.

Reply:
Updated in 9991fcdcfc718d2f6cafb3823f44aa9e31e63210